template/page-xrp-overview.html.jinja

```diff
@@ -181,6 +187,14 @@
               "id": "wallet-ledger",
               "alt": _("Ledger"),
               "imgclasses": "invertible-img" },
+              { "href": "https://keyst.one/",
+              "id": "wallet-keystone",
+              "alt": _("Keystone"),
+              "imgclasses": "invertible-img"},
```

Comment:
```suggestion
              "alt": _("Keystone")},
```

Since you've got a separate logo for Keystone in light mode, and the Keystone logo isn't monochrome, it should not have the `invertible-img` class. As-is, it causes the logo to display with an off-brand orange color in light mode, as in this screenshot:

![2023-08-28_173849_607991492](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/b443060a-7a62-4d04-802d-4223249c6a3f)

The fix I noted in `_light-theme.scss` is also necessary along with this one to get the logo to display correctly.